packages/react/package.json

```
@@ -91,7 +91,7 @@
   ],
   "storybook": {
     "title": "Chakra UI",
-    "url": "https://chakra-ui.netlify.app"
+    "url": "https://chakra-ui-storybook.vercel.app"
```

What do you think about using the subdomain of chakra ui?

```suggestion
    "url": "https://storybook.chakra-ui.com"
```

---

Oh yeah, I didn't know about the subdomain! That's better to avoid future changes! 👍🏻 

Awesome, I'll change this line with your suggestion.